packages/components/forms/src/form-label/FormLabel.tsx

```diff
@@ -0,0 +1,68 @@
+import { cx } from 'emotion';
+import React, { forwardRef } from 'react';
+import type { ReactNode } from 'react';
+import type { PropsWithHTMLElement } from '@contentful/f36-core';
+import { getFormLabelStyles } from './FormLabel.styles';
+import { useFormControl } from '../form-control/FormControlContext';
+import { CommonProps } from '@contentful/f36-core';
+
+export interface FormLabelInternalProps extends CommonProps {
+  /**
+   * Label value to show
+   */
+  children: ReactNode;
+  /**
+   * Whether or not the associated input element is required
+   *
+   * @default false
+   */
+  isRequired?: boolean;
+  /**
+   * Custom text to show in parentheses that gets rendered if the associated
+   * input is required
+   *
+   * @default "required"
+   */
+  requiredText?: string;
+}
+
+export type FormLabelProps = PropsWithHTMLElement<
+  FormLabelInternalProps,
+  'label'
+>;
+
+export const FormLabel = forwardRef<HTMLLabelElement, FormLabelProps>(function (
```

Comment:
We use the `_ComponentName` and exporting the `forwardRef(_ComponentName)` on other components, not a blocker, but maybe to keep it consistent?

Reply:
this is a good idea, I'm going to do that in the follow up PR